source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -508,154 +511,162 @@ namespace AdaptiveCards
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveRichTextBlockRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveRichTextBlockRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveImageRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveImageRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveImageSetRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveImageSetRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveContainerRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveContainerRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveColumnRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveColumnRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveColumnSetRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveColumnSetRenderer();
-            };
-
-            [default_interface] runtimeclass AdaptiveFactRenderer : IAdaptiveElementRenderer
-            {
-                AdaptiveFactRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveChoiceSetInputRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveChoiceSetInputRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveDateInputRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveDateInputRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveNumberInputRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveNumberInputRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveTextInputRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveTextInputRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveTimeInputRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveTimeInputRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveToggleInputRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveToggleInputRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveFactSetRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveFactSetRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveMediaRenderer : IAdaptiveElementRenderer
             {
                 AdaptiveMediaRenderer();
-            };
+            }
 
             [default_interface] runtimeclass AdaptiveActionSetRenderer : IAdaptiveElementRenderer
```

Comment:
>default_interface [](http://example.com/codeflow?start=13&length=17)

@oldnewthing I read [https://devblogs.microsoft.com/oldnewthing/20201028-00/?p=104404](https://devblogs.microsoft.com/oldnewthing/20201028-00/?p=104404) before. I'm wondering if AC made the wrong design on this.

Should we 
```
            [default_interface] runtimeclass AdaptiveMediaRenderer : IAdaptiveElementRenderer
            {
                AdaptiveMediaRenderer();
            };
```
to 
```
             runtimeclass AdaptiveMediaRenderer : default IAdaptiveElementRenderer
            {
                AdaptiveMediaRenderer();
            };
```

Reply:
you might be right..
only **IAdaptiveElementRenderer** should be the default interface here..
@paulcam206, what do you think?